server/routes/coinflip.py

```diff
@@ -35,7 +35,7 @@ def coinflip():
 
         if amount > 0:
             if did_win:
-                tickets_won = amount
+                tickets_won = amount*2
```

Comment:
Why this change?

Reply:
That was the way the original game was said out to be in the writeup so i notieced it wasn't there and changed it.

Reply:
That was the case if playing coinflip takes away the point in the first place. If you want this change, you'll have to change it so that the tickets is deducted every game we play. However, that would be an extra transaction every game and that might be confusing. I say we just remove it.